cli/test/string_empty_to_null_test.dart

```diff
@@ -0,0 +1,15 @@
+import "package:stax/string_empty_to_null.dart";
+import "package:test/test.dart";
+
+void main() {
+  test("Test if empty string returns null", () {
+    String emptyString = "";
+    expect(emptyString.emptyToNull(), null);
+  });
+
+  test("Test if null string returns null", () {
+    String? nullString;
+
+    expect(nullString, null);
```

Comment:
You forgot to call `emptyToNull()` :) 
```suggestion
    expect(nullString.emptyToNull(), null);
```